src/uct/api/v2/uct_v2.h

```diff
@@ -174,6 +177,16 @@ typedef struct {
      * This field is set by the UCT layer.
      */
     ucs_linear_func_t   latency;
+
+    /**
+     * Approximated maximum number of endpoints that could have outstanding
```

Comment:
Is this maximum or just a performance optimal ? I guess you want to avoid transport that have scalability constrains in terms of number of endpoints ?

Reply:
> Is this maximum or just a performance optimal ?

it is maximum number of connections which could be used to send data from some UCT iface.

> I guess you want to avoid transport that have scalability constrains in terms of number of endpoints ?

we will use this attribute on UCP level to select most scalable transport in terms of error detection, e.g. DC transport has only `8` DCIs which could be used simultaneously and they will do packet retransmission until it is successful or "retry count" isn't exceeded - so, these DCIs couldn't be used by other connections (therefore, DC is not good enough for doing keepalive between peers if number of EPs > `8`).
Also, this attribute could be reused for other logics to select transports, if it will be required in the future.

Reply:
I think I would prefer "Approximate" here but am not that "Approximated" isn't acceptable usage either.

Reply:
> I think I would prefer "Approximate" here but am not that "Approximated" isn't acceptable usage either.

done

Reply:
@dmitrygx so, at this point it is only for DC ? Maybe just use some simpler logic saying that the transport does not support keepalive protocol ? You mentioned that you have some other use case for this information. Can you please elaborate ?

Reply:
> @dmitrygx so, at this point it is only for DC ? 

Currently, it is not only for DC. we just discovered that it should be used for RC too - at some scale they couldn't be used for keepalive mechanism and we create separate keepalive lane over UD transport.

> Maybe just use some simpler logic saying that the transport does not support keepalive protocol ? 

They have keepalive in a good shape for small scale, but after some `N` connections detecting peer failures take unacceptable amount of time.

> You mentioned that you have some other use case for this information. Can you please elaborate ?

Currently, we don't have any other use case for `max_inflight_eps`. But we could find new use cases in the future, because it is not connected with keepalive/error-handling functionalities and could be reused for any other functionalities.

Reply:
ok